### PR TITLE
[MRG+1] use ConnectivityMeasure instead of manual fitting of LedoitWolf

### LINKED ISSUE #1211: sklearn.covariance.LedoitWolf or ConnectivityMeasure in plot_adhd_spheres
`ConnectivityMeasure` can be used here, and its default covariance estimator is `LedoitWolf`from `sklearn.covariance`.
I also prefer using partial correlations rather than precision, because no need for negating the connections.

### PR DESCRIPTION
**Pull request by salma1601**
fixes #1211 (and #1214 for `plot_adhd_sphares` example)

### CODE REVIEW

File: examples/03_connectivity/plot_adhd_spheres.py [hunk 1]
```diff
@@ -60,23 +60,23 @@
...
-# Compute precision matrices
-from sklearn.covariance import LedoitWolf
-cve = LedoitWolf()
-cve.fit(time_series)
-
+# Compute partial correlation matrix
+from nilearn.connectome import ConnectivityMeasure
+connectivity_measure = ConnectivityMeasure(kind='partial correlation')
+partial_correlation_matrix = connectivity_measure.fit_transform(
+    [time_series])[0]
```
Comment (bthirion) [inline examples/03_connectivity/plot_adhd_spheres.py]: Is it actually using the LW estimator ? It is at least not explicit.

Comment (salma1601) [inline examples/03_connectivity/plot_adhd_spheres.py]: Yes, `ConnectivityMeasure` default covariance estimator is LW.
May be I can add a comment on that or explicit `cov_estimator` ?

Comment (bthirion) [inline examples/03_connectivity/plot_adhd_spheres.py]: Yes, mark it through an explicit argument or a comment.

**Comment by GaelVaroquaux** [comment]
Thanks! :+1: for merge.

**Pull request merged**